app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryHolder.java

```diff
@@ -167,7 +167,6 @@ private void setAccountNameLayout(AccountNamePosition accountNamePosition) {
             default:
                 profileNameLayoutParams = (RelativeLayout.LayoutParams) _profileName.getLayoutParams();
                 profileNameLayoutParams.addRule(RelativeLayout.END_OF, R.id.profile_issuer);
-                profileNameLayoutParams.setMarginStart(24);
```

Comment:
Keep the margin (instead of the space you introduced), but set it only if there's a non-empty name and issuer.